### PR TITLE
Emissions changes

### PR DESCRIPTION
**Pull request by frahlg**
Changes made:

- Emissions Investors & Founders over 36m
- No stub period. Halvings 2year.
- Emissions added to table.

### CODE REVIEW

File: 0128-energy-subnetwork.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The ENERGY subnetwork is designed to create value for the entire Helium ecosyste
 5. **Sustainability**
    The ENERGY subnetwork aligns Helium with global sustainability goals, potentially attracting environmentally conscious users and partners. This could open up new opportunities for grants, partnerships, and positive public relations for Helium.
 
-The ENERGY subnetwork is a practical expansion for Helium. It builds on the existing network while tapping into the growing renewable energy market, solving the real world problems of the acceleration renewable power. This addition will increase IOT network usage, drive more Data Credit burn, and give Hotspot operators new ways to earn.
+The ENERGY subnetwork is a practical expansion for Helium. It builds on the existing network while tapping into the growing renewable energy market, solving the real world problems of the acceleration renewable power. This addition will increase IoT network usage, drive more Data Credit burn, and give Hotspot operators new ways to earn.
```
Comment (abhay) [inline 0128-energy-subnetwork.md:29]:
```suggestion
The ENERGY subnetwork is a practical expansion for Helium. It builds on the existing network while tapping into the growing renewable energy market, solving the real world problems of the acceleration renewable power. This addition will increase IOT network usage, drive more Data Credit burn, and give Hotspot operators new ways to earn.
```

File: 0128-energy-subnetwork.md [hunk 2]
```diff
@@ -100,16 +100,16 @@ The motivation for the network growth pool is to stimulate growth of the network
...
+| 5               | 202&nbsp;750                        | 11&nbsp;812.5     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
+| 6               | 214&nbsp;562.5                      | 11&nbsp;812.5     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
+| 7               | 226&nbsp;375                        | 5&nbsp;906.25     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
+| …               |                                     |                   |                           |                                |              |                   |         |                 |                  |                      |
```
Comment (abhay) [inline 0128-energy-subnetwork.md:112]:
```suggestion
| Year | ENERGY at the start of the year (M) | ENERGY minted (M) | DER host Baseline Rewards | DER host Service Contract Pool | Sensor hosts | Service Providers | Oracles | Operations Fund | veHNT Delegators | Investors & Founders |
|------|-------------------------------------|-------------------|---------------------------|--------------------------------|--------------|-------------------|---------|-----------------|------------------|----------------------|
| 1    | 1&nbsp;000                          | 47&nbsp;250       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              | 20&nbsp;000          |
| 2    | 68&nbsp;250                         | 47&nbsp;250       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              | 20&nbsp;000          |
| 3    | 135&nbsp;500                        | 23&nbsp;625       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              | 20&nbsp;000          |
| 4    | 179&nbsp;125                        | 23&nbsp;625       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
| 5    | 202&nbsp;750                        | 11&nbsp;812.5     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
| 6    | 214&nbsp;562.5                      | 11&nbsp;812.5     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
| 7    | 226&nbsp;375                        | 5&nbsp;906.25     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |                      |
| …    |                                     |                   |                           |                                |              |                   |         |                 |                  |                      |
```

File: 0128-energy-subnetwork.md [hunk 3]
```diff
@@ -168,7 +168,7 @@ Baseline rewards for an online DER are defined in three buckets based on the fol
 Reward DERs that have been part of the network for a long time;
 
 | Level | Definition                 | Score |
-|-------|----------------------------|-------|
+|-------|--------------------------- |-------|
```
Comment (abhay) [inline 0128-energy-subnetwork.md:171]:
```suggestion
|-------|----------------------------|-------|
```

File: 0128-energy-subnetwork.md [hunk 4]
```diff
@@ -86,9 +86,9 @@ Allocation of ENERGY tokens prioritizes emission to the community of DER hosts f
...
+We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY subnetwork and develop the technical and operational infrastructure required to build the ENERGY subnetwork. These token will be minted over a 36 month period and will have a lockup of a minimum of 12 months after token emission date. 190B ENERGY is emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts, the baseline rewards for participating in the subnetwork and the service rewards for participating in energy services. The baseline rewards will receive 15% of emitted ENERGY and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, remaining tokens are burned. We use net emissions to replenish burned tokens into service rewards using a 30-day average of burned tokens with a cap. The cap will initially be set to 1% of current emissions but will likely be subject to change.
```
Comment (abhay) [inline 0128-energy-subnetwork.md:91]:
```suggestion
We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY subnetwork and develop the technical and operational infrastructure required to build the ENERGY subnetwork. These token will be minted over a 36-month period and will have a lockup of a minimum of 12 months after token emission date. 190B ENERGY is emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts, the baseline rewards for participating in the subnetwork and the service rewards for participating in energy services. The baseline rewards will receive 15% of emitted ENERGY and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, remaining tokens are burned. We use net emissions to replenish burned tokens into service rewards using a 30-day average of burned tokens with a cap. The cap will initially be set to 1% of current emissions but will likely be subject to change.
```

File: 0128-energy-subnetwork.md [hunk 5]
```diff
@@ -86,9 +86,9 @@ Allocation of ENERGY tokens prioritizes emission to the community of DER hosts f
...
+We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY subnetwork and develop the technical and operational infrastructure required to build the ENERGY subnetwork. These token will be minted over a 36-month period and will have a lockup of a minimum of 12 months after token emission date. 190B ENERGY is emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts, the baseline rewards for participating in the subnetwork and the service rewards for participating in energy services. The baseline rewards will receive 15% of emitted ENERGY and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, remaining tokens are burned. We use net emissions to replenish burned tokens into service rewards using a 30-day average of burned tokens with a cap. The cap will initially be set to 1% of current emissions but will likely be subject to change.
```
Comment (jthiller) [inline 0128-energy-subnetwork.md:91]: How do we feel about breaking this up a bit?
```suggestion
We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY subnetwork and develop the technical and operational infrastructure needed for its creation. These tokens will be minted over a 36-month period and will have a minimum lockup of 12 months after the token emission date.

190B ENERGY will be emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts: baseline rewards for participating in the subnetwork and service rewards for participating in energy services. The baseline rewards will receive 15% of emitted ENERGY, and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, with remaining tokens being burned.

The ENERGY protocol will use net emissions to replenish burned tokens into service rewards using a 30-day average of burned tokens, with a cap. The cap will initially be set at 1% of current emissions but will likely be subject to change.
```

**Pull request merged**